Added new FAQs and resolved 'content' labeled FAQs (#137)
conda-incubator/conda-dot-org: learn/faq.md

- **The Choice Between Conda and Mamba**: Conda is the default package manager, while Mamba is a high-performance, drop-in replacement for conda. Mamba offers faster package installations and updates, making it a suitable choice for users who prioritize speed and performance.

Using conda provides a streamlined approach to package management, platform compatibility, environment isolation, and access to an extensive package ecosystem. It is particularly beneficial for data scientists, researchers, and developers working with diverse software requirements across different projects.

## Can I use conda to manage non-Python packages?

Yes, conda can be used to manage non-Python packages. While conda is widely known for its capabilities in managing Python packages, it is not limited to Python alone. Conda supports package management for multiple programming languages, including R, Java, C/C++, and more. It allows you to install, manage, and update packages from different languages, ensuring consistent package installations and dependencies across your projects. 

> To install a non-Python package, you can use the following command:

Reviewer: ```suggestion
To install a non-Python package, you can use the following command:

```

```
conda install <package-name>
```

> By default, Conda will search for the package in the default channels. However, you can also obtain non-Python packages from other channels apart from conda-forge. For example, you can specify a different channel using the -c flag:

Reviewer: ```suggestion

By default, Conda will search for the package in the default channels. However, you can also obtain non-Python packages from other channels apart from conda-forge. For example, you can specify a different channel using the -c flag:

```

```
conda install -c <channel-name> <package-name>
```

> For example, to install a non-Python package like gcc for C/C++ programming language, from the conda-forge channel, you can run:

Reviewer: ```suggestion

For example, to install a non-Python package like gcc for C/C++ programming language, from the conda-forge channel, you can run:

```

```
conda install -c conda-forge gcc
```

> Similarly, you can use conda to manage packages for other languages such as R, Java, or any other supported language. Just replace `<package-name> `with the specific package you want to install.

Reviewer: ```suggestion

Similarly, you can use conda to manage packages for other languages such as R, Java, or any other supported language. Just replace `<package-name> `with the specific package you want to install.
```

- **conda Channels**: conda channels serve as repositories for hosting and managing packages. Channels like [conda-forge](https://conda-forge.org/) offer a wide range of community-maintained packages, expanding the available options for software development and experimentation.

- **The choice between conda and Mamba**: conda is the default package manager, while Mamba is a high-performance, drop-in replacement for conda. Mamba offers faster package installations and updates, making it a suitable choice for users who prioritize speed and performance.

Using conda provides a streamlined approach to package management, platform compatibility, environment isolation, and access to an extensive package ecosystem. It is particularly beneficial for data scientists, researchers, and developers working with diverse software requirements across different projects.

## Can I use conda to manage non-Python packages?

> Yes, conda can be used to manage non-Python packages. While conda is widely known for its capabilities in managing Python packages, it is not limited to Python alone. conda supports package management for multiple programming languages, including R, Java, C/C++, and more. It allows you to install, manage, and update packages from different languages, ensuring consistent package installations and dependencies across the projects. 

Reviewer 2: ```suggestion
Yes, conda can be used to manage non-Python packages. While conda is widely known for its capabilities in managing Python packages, it is not limited to Python alone. Conda supports package management for multiple programming languages, including R, Java, C/C++, and more. It allows you to install, manage, and update packages from different languages, ensuring consistent package installations and dependencies across your projects. 
```

## What is conda-forge?

> [conda-forge](https://conda-forge.org/) is a [community-led GitHub organization](https://github.com/conda-forge) that provides access to thousands of conda package recipes. All of these recipes are open source and can be installed with the conda package manager by specifying conda-forge as the channel.

Reviewer 2: ```suggestion
[Conda-forge](https://conda-forge.org/) is a [community-led GitHub organization](https://github.com/conda-forge) that provides access to thousands of conda package recipes. All of these recipes are open source and can be installed with the conda package manager by specifying conda-forge as the channel.
```

## Why does conda have so many meanings in different contexts?

In different contexts, the term "conda" can refer to various aspects:

> - **conda as a tool**: conda is an open-source, cross-platform package manager used for managing software packages.

Reviewer 2: ```suggestion
- **Conda as a tool**: Conda is an open-source, cross-platform package manager used for managing software packages.
```

> - [**conda packages**](https://docs.conda.io/projects/conda/en/latest/user-guide/concepts/packages.html): These are compressed tarballs or .conda files containing system-level libraries, modules, executables, and metadata, installed into the designated prefix.

Reviewer 2: ```suggestion
- [**Conda packages**](https://docs.conda.io/projects/conda/en/latest/user-guide/concepts/packages.html): These are compressed tarballs or .conda files containing system-level libraries, modules, executables, and metadata, installed into the designated prefix.
```

> - [**conda environments**](https://docs.conda.io/docs/using/envs.html#:~:text=A%20conda%20environment%20is%20a,NumPy%201.6%20for%20legacy%20testing.): conda environments are directories that contain specific collections of conda packages installed for a particular project or use case.

Reviewer 2: ```suggestion
- [**Conda environments**](https://docs.conda.io/docs/using/envs.html#:~:text=A%20conda%20environment%20is%20a,NumPy%201.6%20for%20legacy%20testing.): Conda environments are directories that contain specific collections of conda packages installed for a particular project or use case.
```

> - **conda channels**: conda channels are locations where packages are stored and serve as the base for hosting and managing packages. They are hosted on the [Anaconda.org](https://anaconda.org/) platform.

Reviewer 2: ```suggestion
- **Conda channels**: Conda channels are locations where packages are stored and serve as the base for hosting and managing packages. They are hosted on the [Anaconda.org](https://anaconda.org/) platform.
```

> - **conda, the package format**: conda can also refer to the package format itself, which is an archive containing programs and metadata needed by package managers.

Reviewer 2: ```suggestion
- **Conda, the package format**: Conda can also refer to the package format itself, which is an archive containing programs and metadata needed by package managers.
```

- **Miniconda and Anaconda**: These are distributions that include conda as the package manager. [Miniconda](https://docs.conda.io/en/latest/miniconda.html) provides a minimal installation, while [Anaconda](https://www.anaconda.com/) includes additional pre-installed packages.

> - **conda community**: The conda community includes channels like [conda-forge](https://conda-forge.org/) and [bioconda](https://bioconda.github.io/), which provide additional packages and resources.

Reviewer 2: ```suggestion
- **Conda community**: The conda community includes channels like [conda-forge](https://conda-forge.org/) and [bioconda](https://bioconda.github.io/), which provide additional packages and resources.
```

> - **conda, the tool (Mamba)**: [Mamba](https://mamba.readthedocs.io/en/latest/installation.html) is a high-performance, drop-in replacement for conda, written in C++, offering faster package management.

Reviewer 2: ```suggestion
- **Conda, the tool (Mamba)**: [Mamba](https://mamba.readthedocs.io/en/latest/installation.html) is a high-performance, drop-in replacement for conda, written in C++, offering faster package management.
```

> - [**conda Package Specification**](https://conda.io/projects/conda-build/en/latest/resources/package-spec.html): This refers to the contents of an extracted package, regardless of the compression format used.

Reviewer 2: ```suggestion
- [**Conda Package Specification**](https://conda.io/projects/conda-build/en/latest/resources/package-spec.html): This refers to the contents of an extracted package, regardless of the compression format used.
```

> [Conda-forge](https://conda-forge.org/) is a [community-led GitHub organization](https://github.com/conda-forge) that provides access to thousands of conda package recipes. All of these recipes are open source and can be installed with the conda package manager by specifying conda-forge as the channel.

Reviewer: I know this isn't your addition, Pinak, but we should be sure we're consistent with the conda-forge naming.

@jaimergp, is it supposed to always be lowercase, or is capitalizing at the beginning of a sentence fine?

## Why does conda have so many meanings in different contexts?

In different contexts, the term "Conda" can refer to various aspects:

- **Conda as a tool**: Conda is an open-source, cross-platform package manager used for managing software packages.

> - [**Conda packages**](https://docs.conda.io/projects/conda/en/latest/user-guide/concepts/packages.html): These are compressed tarballs or .conda files containing system-level libraries, modules, executables, and metadata, installed into the designated prefix.

Reviewer: A newer user probably won't know what a prefix is.

> - [**Conda environments**](https://docs.conda.io/docs/using/envs.html#:~:text=A%20conda%20environment%20is%20a,NumPy%201.6%20for%20legacy%20testing.): Conda environments are directories that contain specific collections of Conda packages installed for a particular project or use case.

Reviewer: Are you meaning to have this URL highlight words on the other end?

> - **Conda channels**: Conda channels are locations where packages are stored and serve as the base for hosting and managing packages. They are hosted on the [Anaconda.org](https://anaconda.org/) platform.

Reviewer: Is Anaconda.org the only channel host?

- **Conda, the package format**: Conda can also refer to the package format itself, which is an archive containing programs and metadata needed by package managers.

> - **Miniconda and Anaconda**: These are distributions that include Conda as the package manager. [Miniconda](https://docs.conda.io/en/latest/miniconda.html) provides a minimal installation, while [Anaconda](https://www.anaconda.com/) includes additional pre-installed packages.

Reviewer: There's also Micromamba, though I know you're talking about conda here.

- **Conda community**: The Conda community includes channels like [conda-forge](https://conda-forge.org/) and [bioconda](https://bioconda.github.io/), which provide additional packages and resources.

> - **Conda, the tool (Mamba)**: [Mamba](https://mamba.readthedocs.io/en/latest/installation.html) is a high-performance, drop-in replacement for Conda, written in C++, offering faster package management.

Reviewer: Did you mean to say "Mamba, the tool" here? It's an alternative to conda, but I didn't think you needed conda to use Mamba and this entry kind of makes it seem like you do.

- [**Conda Package Specification**](https://conda.io/projects/conda-build/en/latest/resources/package-spec.html): This refers to the contents of an extracted package, regardless of the compression format used.

- **Installing Conda or packages**: This context refers to the process of installing Conda itself or using Conda to install other packages.

> You can find more about this [here](https://conda.org/community#the-many-meanings-of-conda)

Reviewer: This sentence needs a period at the end.

- **Miniconda and Anaconda**: These are distributions that include conda as the package manager. [Miniconda](https://docs.conda.io/en/latest/miniconda.html) provides a minimal installation, while [Anaconda](https://www.anaconda.com/) includes additional pre-installed packages.

- **Conda community**: The Conda community includes channels like [conda-forge](https://conda-forge.org/) and [bioconda](https://bioconda.github.io/), which provide additional packages and resources.

- [**Mamba**](https://mamba.readthedocs.io/en/latest/installation.html): Mamba is a high-performance, drop-in replacement for Conda. It is written in C++ and offers faster package management.

- [**Conda Package Specification**](https://conda.io/projects/conda-build/en/latest/resources/package-spec.html): This refers to the contents of an extracted package, regardless of the compression format used.

- **Installing conda or packages**: This context refers to the process of installing conda itself or using conda to install other packages.

You can find more about this [here](https://conda.org/community#the-many-meanings-of-conda).

> 
> 

Reviewer: ```suggestion

```

- [**Conda environments**](https://docs.conda.io/projects/conda/en/latest/user-guide/concepts/environments.html): Conda environments are directories that contain specific collections of conda packages installed for a particular project or use case.

- **Conda channels**: Conda channels are locations where packages are stored and serve as the base for hosting and managing packages. They can be hosted on various platforms, including [Anaconda.org](https://anaconda.org/) and others.

- **Conda, the package format**: Conda can also refer to the package format itself, which is an archive containing programs and metadata needed by package managers.

- **Conda distributions**: Conda is included in various distributions. [Miniconda](https://docs.conda.io/en/latest/miniconda.html) provides a minimal installation, [Anaconda](https://www.anaconda.com/) includes additional pre-installed packages, and there is also [Micromamba](https://mamba.readthedocs.io/en/latest/user_guide/micromamba.html), a lightweight distribution based on Conda.

> 
> 

Reviewer: ```suggestion

```

Pip can only install Python packages and (unlike conda) cannot account for the dependency graphs connected to each package that it installs, which can break global system dependencies and/or the user's dependency stacks. Even when using pip with a tool like virtualenv, which creates isolated Python environments, it can still inadvertently install Python packages to the wrong places.

> On the other hand, conda is a powerful package and environment manager that can install much more than just Python libraries. With conda, users can install entire software stacks (while remaining assured that all dependencies are accounted for and resolved), as well as R programs and libraries, Node.js, Java programs, C++ programs and libraries, Perl programs, and more. conda has an environment management system that allows users to have all of these installed across multiple different environments; it also enables installation of complex software stacks on a system without needing root privileges, due to it being able to do all of these software and package installations in an isolated, userspace manner.

Reviewer 2: ```suggestion
On the other hand, conda is a powerful package and environment manager that can install much more than just Python libraries. With conda, users can install entire software stacks (while remaining assured that all dependencies are accounted for and resolved), as well as R programs and libraries, Node.js, Java programs, C++ programs and libraries, Perl programs, and more. Conda has an environment management system that allows users to have all of these installed across multiple different environments; it also enables installation of complex software stacks on a system without needing root privileges, due to it being able to do all of these software and package installations in an isolated, userspace manner.
```

For this reason, it is highly recommended to utilize separate conda environments for each project/purpose in order to mitigate the dependency management issues of the Python packaging system and to keep project dependencies as separate and simple as possible.

## Can I share conda environments with others? If so, how?

Yes, you can share conda environments with others, making it easier to collaborate on projects or reproduce specific software setups. Here's how you can share conda environments:

> 1. **Exporting the environment:** With conda, you can export the environment to a YAML file that contains a list of all the packages and their versions. Open your terminal or command prompt, activate the environment you want to share, and use the following command:

Reviewer: Let's go with "your environment" instead of "the environment" throughout. Makes it a little less formal sounding.

```
conda env export > environment.yml
```
This command exports the environment to an `environment.yml` file.

> 2. **Sharing the environment file:** Once you have the `environment.yml` file, you can share it with others through various means, such as email, file sharing services, or version control systems like Git. Make sure to communicate any specific instructions or requirements related to the environment, such as additional dependencies or channels.

Reviewer: Doesn't the environment.yml file include information about dependencies in it?

3. **Creating the environment from the file:** To create an identical environment on another machine, the other person can use the following command in their terminal or command prompt:
```
conda env create -f environment.yml
```

> This command reads the `environment.yml` file and recreates the environment with the same package versions and dependencies. The person needs to have conda or mamba installed on their system.

Reviewer: Yeah, go with "you" instead of "the person". We tend to use 2nd-person throughout our writing to decrease the formality.

Note: If using mamba instead of conda, the command will be:
```
mamba env create -f environment.yml
```
4. **Activating the shared environment:** After the environment is successfully created, the other person can activate it with:
```
conda activate <environment-name>
```
or
```
mamba activate <environment-name>
```

> Replace `<environment-name>` with the name of the shared environment.

Reviewer: There should be a newline after this line. Otherwise, it will end up in the same paragraph as the line below.

By following these steps, you can easily share conda environments, ensuring consistent dependencies and versions.

> Remember that channels play a crucial role in environment replication. If the environment relies on packages from specific channels other than the defaults, the other person might need to add those channels using:

Reviewer: "If your environment relies on..., you might need to..."

```
conda config --add channels <channel-name>
```
or 
```
mamba config --add channels <channel-name>
``` 
before creating the environment from the YAML file.

## I want to create a conda package for my project. Where do I start?

To create a conda package for your project, you can start by referring to the official documentation provided by [conda-build](https://github.com/conda/conda-build).

> For step-by-step guidance on setting up your package, defining dependencies, handling different platforms, and more, you can checkout the [documentation for conda-build](https://docs.conda.io/projects/conda-build/en/stable/user-guide/tutorials/building-conda-packages.html) .

Reviewer: "check out" should be two words here

Reviewer: And you've got a space between the last word and the period at the end of the sentence that shouldn't be there.

> If you need further assistance or have specific questions related to creating conda packages, I recommend visiting the [conda community page](https://conda.org/community).

Reviewer: Putting "I" in here when it's not a blog is a little _too_ formal. Just change "I recommend visiting" to "visit".

For this reason, it is highly recommended to utilize separate conda environments for each project/purpose in order to mitigate the dependency management issues of the Python packaging system and to keep project dependencies as separate and simple as possible.

## Can I share conda environments with others? If so, how?

Yes, you can share conda environments with others, making it easier to collaborate on projects or reproduce specific software setups. Here's how you can share conda environments:
1. **Exporting the environment:** With conda, you can export the environment to a YAML file that contains a list of all the packages and their versions. Open the terminal or command prompt, activate the environment you want to share, and use the following command:
```
conda env export > environment.yml
```

> This command exports the environment to an `environment.yml` file.

Reviewer: ```suggestion

This command exports the environment to an `environment.yml` file.
```

2. **Sharing the environment file:** Once you have the `environment.yml` file, you can share it with others through various means, such as email, file sharing services, or version control systems like Git. 

The `environment.yml` file contains information about the dependencies required for the environment, including packages, versions, and channels. Make sure to communicate any specific instructions or requirements related to the environment, such as additional dependencies or channels.

> 
> 

Reviewer: ```suggestion

```

> 3. **Creating the environment from the file:** To create an identical environment on another machine, the other person can use the following command in their terminal or command prompt:

Reviewer: ```suggestion
3. **Creating the environment from the file:** To create an identical environment on another machine, the other person can use the following command in their terminal or command prompt:

```

```
conda env create -f environment.yml
```

> This command reads the `environment.yml` file and recreates the environment with the same package versions and dependencies. The person building an environment from this file needs to have conda or mamba installed on their system.

Reviewer: ```suggestion

This command reads the `environment.yml` file and recreates the environment with the same package versions and dependencies. The person building an environment from this file needs to have conda or mamba installed on their system.
```

Note: If using mamba instead of conda, the command will be:
```
mamba env create -f environment.yml
```

> 4. **Activating the shared environment:** After the environment is successfully created, the other person can activate it with:

Reviewer: ```suggestion

4. **Activating the shared environment:** After the environment is successfully created, the other person can activate it with:

```

```
conda activate <environment-name>
```

> or

Reviewer: ```suggestion

or

```

```
mamba activate <environment-name>
```

> Replace `<environment-name>` with the name of the shared environment.

Reviewer: ```suggestion

Replace `<environment-name>` with the name of the shared environment.
```

By following these steps, you can easily share conda environments, ensuring consistent dependencies and versions.

> Remember that channels play a crucial role in environment replication. If your environment relies on packages from specific channels other than the defaults, then you might need to add those channels using:

Reviewer: ```suggestion
Remember that channels play a crucial role in environment replication. If your environment relies on packages from specific channels other than the defaults, then you might need to add those channels using:

```

```
conda config --add channels <channel-name>
```

> or 

Reviewer: ```suggestion

or

```

```
mamba config --add channels <channel-name>
``` 

> before creating the environment from the YAML file.

Reviewer: ```suggestion

before creating the environment from the YAML file.
```

## Can I share conda environments with others? If so, how?

Yes, you can share conda environments with others, making it easier to collaborate on projects or reproduce specific software setups. Here's how you can share conda environments:

> 1. **Exporting the environment:** With conda, you can export the environment to a YAML file that contains a list of all the packages and their versions. Open the terminal or command prompt, activate the environment you want to share, and use the following command:

Reviewer: ```suggestion

1. **Exporting the environment:** With conda, you can export the environment to a YAML file that contains a list of all the packages and their versions. Open the terminal or command prompt, activate the environment you want to share, and use the following command:

```

```
conda env export > environment.yml
```
This command exports the environment to an `environment.yml` file.

2. **Sharing the environment file:** Once you have the `environment.yml` file, you can share it with others through various means, such as email, file sharing services, or version control systems like Git. 

The `environment.yml` file contains information about the dependencies required for the environment, including packages, versions, and channels. Make sure to communicate any specific instructions or requirements related to the environment, such as additional dependencies or channels.

3. **Creating the environment from the file:** To create an identical environment on another machine, the other person can use the following command in their terminal or command prompt:
```
conda env create -f environment.yml
```
This command reads the `environment.yml` file and recreates the environment with the same package versions and dependencies. The person building an environment from this file needs to have conda or mamba installed on their system.

> Note: If using mamba instead of conda, the command will be:

Reviewer: ```suggestion

Note: If using mamba instead of conda, the command will be:

```

```
conda env create -f environment.yml
```

> This command reads the `environment.yml` file and recreates the environment with the same package versions and dependencies. You need to have conda or mamba installed on their system.

Reviewer 2: ```suggestion
This command reads the `environment.yml` file and recreates the environment with the same package versions and dependencies. The person building an environment from this file needs to have conda or mamba installed on their system.
```

## I want to create a conda package for my project. Where do I start?

> To create a conda package for the project, you can start by referring to the official documentation provided by [conda-build](https://github.com/conda/conda-build).

Reviewer: ```suggestion
To create a conda package for your project, you can start by referring to the official documentation provided by [conda-build](https://github.com/conda/conda-build).
```

Your is fine here. :)

> For step-by-step guidance on setting up the package, defining dependencies, handling different platforms, and more, you can check out the [documentation for conda-build](https://docs.conda.io/projects/conda-build/en/stable/user-guide/tutorials/building-conda-packages.html).

Reviewer: ```suggestion
For step-by-step guidance on setting up packages, defining dependencies, handling different platforms, and more, you can check out the [documentation for conda-build](https://docs.conda.io/projects/conda-build/en/stable/user-guide/tutorials/building-conda-packages.html).
```

> If you need further assistance or have specific questions related to creating conda packages, visit [conda community page](https://conda.org/community).

Reviewer: ```suggestion
If you need further assistance or have specific questions related to creating conda packages, visit the [conda community page](https://conda.org/community).
```

See our [Contributing](https://github.com/conda-incubator/conda-dot-org/blob/main/CONTRIBUTING.md) documentation for more information on submitting blog posts.

## How is the conda community organized?

> The Conda community is a vibrant and diverse group of users and developers who utilize the Conda package management system for creating, managing, and distributing software environments. The community is organized through various channels to facilitate collaboration and support.

Reviewer: Not just conda.

See our [Contributing](https://github.com/conda-incubator/conda-dot-org/blob/main/CONTRIBUTING.md) documentation for more information on submitting blog posts.

> ## How is the Conda community organized?

Reviewer: ```suggestion
## How is the conda community organized?
```

> The Conda community is a vibrant and diverse group of users and developers who utilize Conda, a popular package management system, for creating, managing, and distributing software environments. The Conda community consists of various sub-communities that play an important role in supporting and extending the functionality of Conda. Some of these communities include:

Reviewer: ```suggestion
The conda community is a vibrant and diverse group of users and developers who utilize conda, a popular package management system, for creating, managing, and distributing software environments. The conda community consists of various sub-communities that play an important role in supporting and extending the functionality of conda. Some of these communities include:
```

> - [Conda Forge](https://conda-forge.org/): A community-led collection of recipes, build infrastructure, and packages for Conda. It provides a wide range of community-maintained packages that can be easily installed using Conda.

Reviewer: ```suggestion
- [conda-forge](https://conda-forge.org/): A community-led collection of recipes, build infrastructure, and packages for conda. It provides a wide range of community-maintained packages that can be easily installed using conda.
```

> - [Bioconda](https://bioconda.github.io/): A specialized community for bioinformatics software packages. Bioconda offers a comprehensive collection of bioinformatics tools and libraries that can be easily managed with Conda.

Reviewer: ```suggestion
- [Bioconda](https://bioconda.github.io/): A specialized community for bioinformatics software packages. Bioconda offers a comprehensive collection of bioinformatics tools and libraries that can be easily managed with conda.
```

> - [Micromamba](https://mamba.readthedocs.io/en/latest/user_guide/micromamba.html): A lightweight, fast, and pure C++based alternative to Conda. Micromamba is designed to provide a streamlined package management experience and is compatible with Conda environments and packages.

Reviewer: ```suggestion
- [Micromamba](https://mamba.readthedocs.io/en/latest/user_guide/micromamba.html): A lightweight, fast, and pure C++ based alternative to conda. Micromamba is designed to provide a streamlined package management experience and is compatible with conda environments and packages.
```

> These communities actively contribute to the Conda ecosystem, providing additional packages, resources, and support for users and developers.

Reviewer: ```suggestion
These communities actively contribute to the conda ecosystem, providing additional packages, resources, and support for users and developers.
```

- [Conda Forge](https://conda-forge.org/): A community-led collection of recipes, build infrastructure, and packages for Conda. It provides a wide range of community-maintained packages that can be easily installed using Conda.

- [Bioconda](https://bioconda.github.io/): A specialized community for bioinformatics software packages. Bioconda offers a comprehensive collection of bioinformatics tools and libraries that can be easily managed with Conda.

- [Micromamba](https://mamba.readthedocs.io/en/latest/user_guide/micromamba.html): A lightweight, fast, and pure C++based alternative to Conda. Micromamba is designed to provide a streamlined package management experience and is compatible with Conda environments and packages.

These communities actively contribute to the Conda ecosystem, providing additional packages, resources, and support for users and developers.

> 
> 
> To contact us, please refer to the [Community Page](https://conda.org/community#how-to-reach-us).

Reviewer: ```suggestion
To contact us, please refer to the [Community Page](https://conda.org/community#how-to-reach-us).
```

## Does the conda community have online events, meetings, or calls?

The conda community offers diverse events and meetings for engagement:

- **Anaconda** hosts webinars, tutorials, and industry events. Check out the list of upcoming and archived events [here](https://www.anaconda.com/events).

- Regular meetings are conducted by the **conda community** to discuss updates and ideas. [Find the meeting notes here](https://hackmd.io/@conda-community?tags=%5B%22meeting-notes%22%5D).

- **Conda-forge** holds biweekly developer meetings, with [minutes available here](https://conda-forge.org/docs/orga/minutes/00_intro.html).

- **Mamba** also organizes biweekly developer meetings. [Get more information here](https://github.com/mamba-org/mamba#biweekly-dev-meeting).

Participating in these events and meetings provides valuable opportunities to connect and contribute to the conda community.

> 
> 

Reviewer: ```suggestion

```
